packages/client/src/helpers/DynascaleManager.ts

```diff
@@ -413,7 +427,7 @@ export class DynascaleManager {
           .subscribe((isPublishing) => {
             if (isPublishing) {
               // the participant just started to publish a track
-              requestTrackWithDimensions(DebounceType.FAST, {
+              requestTrackWithDimensions(DebounceType.IMMEDIATE, {
                 width: videoElement.clientWidth,
                 height: videoElement.clientHeight,
               });
```

Comment:
For this case (subscribing after a participant started to publish video), do we even need debouncing? I think it would be nice to just subscribe immediately.